packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -38,20 +38,12 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
     /// @param startDate The timestamp when the proposal starts.
     /// @param endDate The timestamp when the proposal expires.
     struct ProposalParameters {
-        uint256 minApprovals;
+        uint16 minApprovals;
```

Comment:
Why not uint32 or uint64?  65k approvals (16 bit) is more than probably needed, but we're not going to add any struct field in the gap between these two, I guess.